### ISSUE TITLE
Training loss chart

### ISSUE THREAD

**Issue by pushgct** [AlexeyAB/darknet#4633]
how do I get this training loss chart for my own custom dataset pls?
I have trained using below command in Google Colab and don't get any useful output:

!./darknet detector train -map data/bdd100k.data cfg/yolov3-bdd100k.cfg yolov3.weights -dont_show

 105 conv     45       1 x 1/ 1     52 x  52 x 256 ->   52 x  52 x  45 0.062 BF
 106 yolo
[yolo] params: iou loss: mse (2), iou_norm: 0.75, cls_norm: 1.00, scale_x_y: 1.00
Total BFLOPS 65.355 
 Allocate additional workspace_size = 49.84 MB 
Loading weights from yolov3.weights...
 seen 64 
Done! Loaded 107 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Saving weights to backup//yolov3-bdd100k_final.weights

**Comment by AlexeyAB** [comment]
Read how to train: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

**Comment by pushgct** [comment]
> Read how to train: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

I did all steps in the link you provided. but it doesn't generate any trained weights in backup directory or doesn't give loss chart too. I get only the above output.

**Comment by AlexeyAB** [comment]
There is not written that you should use file yolov3.weights for training

**Comment by pushgct** [comment]
> There is not written that you should use file yolov3.weights for training

which weights should I choose for training for yolov3 please? I want to train yolov3 for my custom dataset.

**Comment by AlexeyAB** [comment]
https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

> Start training by using the command line: darknet.exe detector train data/obj.data yolo-obj.cfg darknet53.conv.74

**Comment by pushgct** [comment]
> https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects
> 
> > Start training by using the command line: darknet.exe detector train data/obj.data yolo-obj.cfg darknet53.conv.74

Can I replace the obj.data file to point to my training images? train.txt? and number of classes to my .names files?

**Comment by AlexeyAB** [comment]
https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects

> Create file obj.data in the directory build\darknet\x64\data\, containing (where classes = number of objects):
> 
> classes= 2
> train  = data/train.txt
> valid  = data/test.txt
> names = data/obj.names
> backup = backup/

**Comment by pushgct** [comment]
> https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects
> 
> > Create file obj.data in the directory build\darknet\x64\data, containing (where classes = number of objects):
> > classes= 2
> > train  = data/train.txt
> > valid  = data/test.txt
> > names = data/obj.names
> > backup = backup/

I have to run Darknet from /darknet/ home directory or from darknet/build/darknet/x64/ directory pls? As there is no Makefile inside darknet/build/darknet/x64/ directory.

Also does the .data and .cfg file names have to be obj.data and yolo-obj.cfg please?
